continuous_integration/recipe/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev') + environ.get('VERSION_SUFFIX', '') %}
+{% set py_version=environ.get('CONDA_PY', 36) %}
+
+
+package:
+  name: distributed
+  version: {{ version }}
+
+source:
+  git_url: ../..
+
+build:
+  number: {{ GIT_DESCRIBE_NUMBER }}
+  string: py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - dask-scheduler = distributed.cli.dask_scheduler:go
+    - dask-ssh = distributed.cli.dask_ssh:go
+    - dask-worker = distributed.cli.dask_worker:go
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+  host:
+    - python
+    - pip
+
+  run:
+    - python
+    - click >=6.6
+    - cloudpickle >=1.5.0
+    - cytoolz >=0.8.2
+    - dask-core {{ version }}
```

Comment:
How should we pin the `dask-core` dependency for pre-releases? Ideally we would want a Distributed pre-release running with the most recently published `dask-core` pre-release at the time of publishing, but this doesn't seem possible by pinning to the Distributed nightly version, as there will be cases where there isn't a matching `dask-core` pre-release (i.e. the latest `dask-core` pre-release is over a day old).

Reply:
Ended up pinning `dask-core` to the latest available pre-release version / build